packages/asl-projects/client/pages/sections/protocols/protocol-sections.js

```diff
@@ -105,9 +105,9 @@ class ProtocolSections extends PureComponent {
         }
         <Expandable expanded={this.state.expanded} onHeaderClick={this.toggleExpanded}>
           <Completable status={values.deleted ? 'deleted' : values.complete ? 'complete' : 'incomplete'}>
-            <h2 className="title inline-block">{values.deleted ? title : `${number + 1}: ${title}`}</h2>
+            <button className='govuk-button no-bg'><h2 className="title inline-block">{values.deleted ? title : `${number + 1}: ${title}`}</h2></button>
```

Comment:
```suggestion
            <button className='govuk-button link'><h2 className="title inline-block">{values.deleted ? title : `${number + 1}: ${title}`}</h2></button>
```

This is what was tried in the demo - what issues are you seeing when trying that?

Reply:
<img width="603" height="348" alt="Screenshot 2025-10-29 at 1 44 55 pm" src="https://github.com/user-attachments/assets/691676c0-290f-4be7-9bfa-7baaac0b8180" />